src/platform/tigerlake/include/platform/lib/memory.h

```diff
@@ -372,6 +372,7 @@
 #define SOF_MEM_VECT_TEXT_SIZE		0x38
 #define SOF_MEM_VECT_SIZE		(SOF_MEM_VECT_TEXT_SIZE + \
 					SOF_MEM_VECT_LIT_SIZE)
+#define DYNAMIC_VECTORS_SIZE		0x400
```

Comment:
prefix needed, plus it could do with a comment describing what its used for.

Reply:
done